core/kazoo_modb/src/kazoo_modb.erl

```diff
@@ -272,11 +272,12 @@ maybe_create_current_modb(<<"account%2F", AccountId/binary>>) ->
     maybe_create_current_modb(binary:replace(AccountId, <<"%2F">>, <<>>, ['global'])).
 
 -spec create(ne_binary()) -> 'ok'.
-create(?MATCH_MODB_SUFFIX_RAW(AccountId, _, _) = AccountMODb) ->
```

Comment:
i don't like this, as in #2643 and #2644 , if the problem is encoded modb name, add a function clause.